lib/modules/manager/bundler/extract.ts

```diff
@@ -103,11 +104,42 @@ export async function extractPackageFile(
       });
     }
 
+    const gemGitRefsMatchRegex = regEx(
+      `^\\s*gem\\s+(['"])(?<depName>[^'"]+)['"]((\\s*,\\s*git:\\s*['"](?<gitUrl>[^'"]+)['"])|(\\s*,\\s*github:\\s*['"](?<repoName>[^'"]+)['"]))(\\s*,\\s*branch:\\s*['"](?<branchName>[^'"]+)['"])?(\\s*,\\s*ref:\\s*['"](?<refName>[^'"]+)['"])?(\\s*,\\s*tag:\\s*['"](?<tagName>[^'"]+)['"])?`,
+    );
+    const gemGitRefsMatch = gemGitRefsMatchRegex.exec(line);
```

Comment:
```suggestion
    const gemGitRefsMatch = gemGitRefsMatchRegex.exec(line)?.groups;
```
makes following code easier